plugins/woocommerce/client/admin/client/settings-payments/components/sortable/sortable-container.tsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import {
+	closestCenter,
+	DndContext,
+	KeyboardSensor,
+	MouseSensor,
+	TouchSensor,
+	useSensor,
+	useSensors,
+	DragEndEvent,
+	DragStartEvent,
+} from '@dnd-kit/core';
+import {
+	SortableContext,
+	verticalListSortingStrategy,
+	horizontalListSortingStrategy,
+	arrayMove,
+} from '@dnd-kit/sortable';
+import {
+	restrictToHorizontalAxis,
+	restrictToVerticalAxis,
+} from '@dnd-kit/modifiers';
+import clsx from 'clsx';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './sortable.scss';
+
+export const SortableContainer = < T extends { id: string } >( {
+	items,
+	setItems,
+	children,
+	sorting = 'vertical',
+	onDragStart = () => {},
+	onDragEnd = () => {},
+	className = '',
+}: {
+	items: T[];
+	setItems: ( items: T[] ) => void;
+	children: React.ReactNode;
+	sorting?: 'vertical' | 'horizontal';
+	onDragStart?: ( event: DragStartEvent ) => void;
+	onDragEnd?: ( event: DragEndEvent ) => void;
+	className?: string;
+} ) => {
+	const sensors = useSensors(
+		useSensor( MouseSensor, {} ),
+		useSensor( TouchSensor, {} ),
+		useSensor( KeyboardSensor, {} )
```

Comment:
I realised I could use the keyboard to manipulate the order thanks to this, by tabbing until the 'enable' button is highlighted then shift-tabbing once, and tapping space followed by arrow keys:
<img width="837" alt="image" src="https://github.com/user-attachments/assets/990db4cd-c46b-4a1a-b8b6-19a4a4f4a15d">

It'd be amazing if we could add the focus highlight to add the cherry on top!

Reply:
Nice find! Tbh I didn't know about this, the sensors were part of a boilerplate 😅 

Anyway, I've added the focus for keyboard in a413cbc

<img width="372" alt="image" src="https://github.com/user-attachments/assets/a547e81c-7ddc-49a6-9f2d-cfb3ba0cdec6">